src/Hedgehog/Range.fs

```diff
@@ -212,8 +222,9 @@ module Range =
     /// >>> Range.bounds 99 <| Range.linear 0 10
     /// (0, 10)
     ///
-    let inline linear (x : 'a) : ('a -> Range<'a>) =
-      linearFrom x x
+    [<CompiledName("Linear")>]
+    let inline linear (x : 'a) (y : 'a) : Range<'a> =
+      linearFrom x x y
```

Comment:
What's the motivation for this change? 😃 The idea here is that this, ported, code tries to be as close as possible with the Haskell version; we even ported the doctests 🐻